js/build.js

```diff
@@ -98,7 +98,8 @@ function reDrawAllSlides(data, widgetId) {
 
 // Main fanction to update and show slides
 function updateSlide(data, widgetId, activeSlide) {
-  var $slidesInDom = $('[data-onboarding-id='+widgetId+'] .swiper-container [data-slider-id]');
+  // When we open interface we have 2 data-onboarding-id elements that's why we take only first of them.
```

Comment:
@YaroslavOvdii Have we looked into why there are 2 elements with the same `data-onboarding-id`, and perhaps take one out? Having 2 on the page just adds more complication for future work.

Reply:
@tonytlwu 
The second element appears only when we open the interface. And duplicated element has this class `fl-widget-instance fl-mirror-element fl-mirror-root`.
May this mirror element being added from the studio?

Reply:
Ah yes, the mirror element. Hmm, it's a long story. Let's keep your solution as it is then. We're working on a big overhaul in the Studio interface which will get rid of the mirror elements.